### PR TITLE
error fixes and graph modification

### PR DESCRIPTION
**Pull request by Kevoulee**
Dear @hansieodendaal please see changes to initial poc file which includes error fixes and changes to the graph format

### CODE REVIEW

File: scenarios/initial_poc.py [hunk 1]
```diff
@@ -84,18 +74,22 @@
 
 P_tot = []
```
Comment (hansieodendaal) [inline scenarios/initial_poc.py:75]:
```suggestion
P_tot = []
print('Tot_nodes Bad_nodes Committee Threshold  index   Probability to control')
```

File: scenarios/initial_poc.py [hunk 2]
```diff
@@ -84,18 +74,22 @@
...
 for j in range(0, len(no_of_nodes)):
-    P_tot.append(hdp.probability(bft_threshold, no_of_bad_actors, committee_size, no_of_nodes))
+    P_tot.append(hdp.probability(bft_threshold[j], no_of_bad_actors[j], committee_size[j], no_of_nodes[j]))
     print(' %3s  %3s  %3s  %3s    j=%3s     P_tot= %-20s' % \
```
Comment (hansieodendaal) [inline scenarios/initial_poc.py]:
```suggestion
    print('    %3s      %3s        %3s     %3s      j=%3s   P_tot= %-20s' % \
```

**Review by hansieodendaal — DISMISSED** [review]
Looking good!
Suggest add headings for column output as indicated.

**Comment by Kevoulee** [comment]
@hansieodendaal, thank you so much, I have committed changes.

**Review by hansieodendaal — APPROVED** [review]

**Pull request merged**